docs/manuals/gui/corelements/job_selector.md_template

```diff
@@ -1,8 +1,124 @@
+  <img src="../job_selector-filter-d.png" class="visible-dark"  />
+  <img src="../job_selector-filter-l.png" class="visible-light" />
+  <figcaption>The filter button</figcaption>
+</figure>
+
+When the user clicks that button, a filter dialog pops up where one can set different filters with
+different criteria:
+<figure class="tp-center">
+  <img src="../job_selector-filter1-d.png" class="visible-dark"  width="80%"/>
+  <img src="../job_selector-filter1-l.png" class="visible-light" width="80%"/>
+  <figcaption>The filter dialog</figcaption>
+</figure>
+
+Initially, this dialog has no configured filter, therefore the *apply* button indicates that '0'
+filters would be set.
+
+To create a new filter, the user must select one of the control's columns, an operator (such as
+*is* and *is not*), and an operand as a value.<br/>
+In the following image, the user has indicated that only the jobs where the *Status* is set to
+"ABANDONED" should be listed:
+<figure class="tp-center">
+  <img src="../job_selector-filter2-d.png" class="visible-dark"  width="80%"/>
+  <img src="../job_selector-filter2-l.png" class="visible-light" width="80%"/>
+  <figcaption>Specifying a filter</figcaption>
+</figure>
+
+After the 'Add filter' button (with the '+' sign) is pressed, the filter is created:
+<figure class="tp-center">
+  <img src="../job_selector-filter3-d.png" class="visible-dark"  width="80%"/>
+  <img src="../job_selector-filter3-l.png" class="visible-light" width="80%"/>
+  <figcaption>Adding a filter</figcaption>
+</figure>
+
+The filter can be removed by pressing the trash icon to the right of the filter row.
+
+Note that the 'Apply' button now indicates that there is a filter that can be applied.
+
+Pressing the 'Apply' button applies the filter to the job selector list, resulting in the following
+image:
+<figure class="tp-center">
+  <img src="../job_selector-filter4-d.png" class="visible-dark"  />
+  <img src="../job_selector-filter4-l.png" class="visible-light" />
+  <figcaption>The filter is applied</figcaption>
+</figure>
+
+Note that next to the 'filter' button, the control now displays the number of currently applied
+filters.
+
+# Usage
+
+## Show or hide columns
+
+The control's properties [*show_id*](#p-show_id),
+[*show_submitted_label*](#p-show_submitted_label), [*show_submitted_id*](#p-show_submitted_id),
+[*show_submission_id*](#p-show_submission_id), [*show_date*](#p-show_date),
+[*show_cancel*](#p-show_cancel), and [*show_delete*](#p-show_delete), and
+[*show_submit_sequences*](#p-show_submit_sequences) let you indicate what columns the job
+selector control displays.
+
+We can define our control so the job identifiers and the submission dates are not shown:
+!!! example "Page content"
+
+    === "Markdown"
+
+        ```
+        <|{job}|job_selector|don't show_id|don't show_date|>
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:job_selector show_id="false" show_date"false">{job}</taipy:job_selector>
```

Comment:
not sure about this prefix ...

Reply:
What do you mean?

Reply:
did you try <taipy:job_selector ... ?
should it be <taipy_gui_core:job_selector ... ?

Reply:
Got it.